src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).get();
```

Comment:
let's throw an exception in case if optional is null

Reply:
But we already throw an exception in DAO layer in case of impossibility to fetch an entity from db, so, we just wouldn't even get to return statement in DAO, thus, CinemaHallDao.get(id).get() in Service layer there will never return null.
And, btw, throwing exception in Service doesn't sound good.

Reply:
We discussed this issue in previous homework with Anastasia Len, you approved the PR. :-)